hack/tilt/kubetail.yaml

```diff
@@ -525,28 +508,49 @@ spec:
               readOnly: true
             - name: varlog
               mountPath: /var/log
+              readOnly: true
             - name: varlibdockercontainers
               mountPath: /var/lib/docker/containers
               readOnly: true
           readinessProbe:
-            grpc:
-              port: 50051
+            exec:
+              command:
+                - /bin/grpc_health_probe
```

Comment:
Hey @amorey as I'll look into implementing this for the new cluster agent, can you clarify if I understood this change correctly?

So previously we were using the native k8s gRPC health checking but now we moved to using this tool in order to enable mTLS? So the idea is that k8s will execute this command in the cluster agent's pod which in turn will check if the local gRPC service is enabled?